org.eu.encom.spectral.yaml

```diff
@@ -0,0 +1,23 @@
+id: org.eu.encom.spectral
+runtime: org.kde.Platform
+runtime-version: 5.11
+sdk: org.kde.Sdk
+command: spectral
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --device=dri
+  - --filesystem=home
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
```

Comment:
Have you tested this works, I know other Qt apps have problems with it.

Reply:
Yeah this works.